controllers/comments.controller.js

```diff
@@ -8,7 +8,17 @@ exports.postComment = async (req, resp, next) => {
     article_id,
   };
   insertComment(comment)
-    .then(async (data) => {
+    .then((data) => {
+      if (data.code === "22P02") {
+        resp.status(400).send({ msg: "Invalid ID" });
+      }
+      if (data.code === "23503") {
+        const table = data.detail.split(" ").at(-1);
+        if (table === '"users".') {
+          resp.status(404).send({ msg: "User not found" });
+        }
+        resp.status(404).send({ msg: "Article not found" });
+      }
```

Comment:
This seems to be to do with errors, so should be handled in your PSQL error controller, not this one